.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Proved.

![screen shot 2016-02-16 at 19 21 05](https://cloud.githubusercontent.com/assets/968014/13086210/87faa8f4-d4e2-11e5-82dc-3a4bb4dd7bfd.png)

I see that everywhere: terminal, gitup, atom, sourcetree.
If I add `.gitignore` then it gets hidden.